RELEASE/scripts/autoscend.ash

```diff
@@ -2783,37 +2777,74 @@ boolean LX_getDigitalKey()
 
 int auto_freeCombatsRemaining()
 {
+	return auto_freeCombatsRemaining(false);
+}
+
+int auto_freeCombatsRemaining(boolean log_for_debug)
+{
+
+	void debugPrint(string msg)
+	{
+	  if (!log_for_debug) return;
+	  print(msg, "red");
+	}
```

Comment:
Use `auto_log_debug()` instead. Then you don't need the parameter.

Reply:
Thank you. The idea I had was that if an error is detected, then it specifically calls this count with the parameter to trigger a count. Even if we are not on debug.

Would the follow be an appropriate alternative to what I did above?
```
if(error)
{
  backupSetting("auto_logLevel", debug);
  auto_freeCombatsRemaining();
  restoreSetting("auto_logLevel");
```

Reply:
Oh, jeparo's suggestion below looks better than my suggested code change above.

Reply:
made jeparo's suggested change. Please let me know if you still think I should fix it in light of what jeparo said.

Reply:
All good now :+1: